packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -81,6 +82,15 @@ export const BootstrapCommand: ICommand = {
       ms: Date.now() - packageStartTime,
     });
 
+    if (process.platform === 'darwin' && process.arch === 'arm64') {
+      const patchNativeModulesStartTime = Date.now();
+      await patchNativeModulesForArmMacs(log, kibanaProjectPath);
+      timings.push({
+        id: 'patch native modudles for arm macs',
+        ms: Date.now() - patchNativeModulesStartTime,
+      });
+    }
```

Comment:
@mistic I placed this here because `//packages:build` potentially installs new versions of these packages, but do you think there's a way we could use a babel run command or something to execute this logic after the specific node modules are updated and before the packages are built? Right now none of the package builds require these packages but I could imagine `node-sass` being needed relatively soon by something. Thoughts?